tools/android/packaging/xbmc/build.gradle.in

```diff
@@ -2,6 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion @TARGET_SDK@
+    ndkPath "/home/jenkins/android-tools/android-ndk-r21e"
```

Comment:
no hardcoded paths please, this will fail on the jenkins docker pipelines

Reply:
Yes I know, it's just a test to see if it works with the path since it doesn't seem to resolve `System.getenv("NDK_PATH")`. In local it works.

One doubt, the dockers are in `Release` build mode, the previous ones were in `RelWithDebInfo' .